Improve: Busted installation instructions on Windows

RollanzMushing opened the pull request:
#### Brief overview of PR changes/additions
- added instructions to install LuaRocks and Busted to be compatible with Mudlet
- added instructions to work around the package pathing problem
#### Motivation for adding to Mudlet
- getting Busted to work with Mudlet on Windows is non-trivial
- suggested by Vadi

RollanzMushing committed:
Testing instructions for Windows

vadi2 commented on src/mudlet-lua/tests/README.md:
```diff
@@ -9,13 +9,33 @@ On **macOS**:
+	
+On **Windows**
+
+1. Install LuaRocks (Windows all-in-one executable (32-bit) is the version you need)
+	- Unzip the installer
```
```suggestion
1. Download and unzip [LuaRocks](https://luarocks.org/releases/luarocks-3.8.0-windows-32.zip)

```

vadi2 commented on src/mudlet-lua/tests/README.md:
```diff
@@ -9,13 +9,33 @@ On **macOS**:
+	  - Setting `Path` and `PATHEXT` is fine
+2. Install Busted
+	- Open a command prompt and enter `luarocks install busted`
+	- If you get a command not recognized message
```
```suggestion
	- If you get a `command not recognized` message:
```

RollanzMushing replied:
I have changed the text to more accurately reflect the message that would appear.

vadi2 commented on src/mudlet-lua/tests/README.md:
```diff
@@ -9,13 +9,33 @@ On **macOS**:
+
+1. Install LuaRocks (Windows all-in-one executable (32-bit) is the version you need)
+	- Unzip the installer
+  - Open the x86 Native Tools Command Prompt that comes with Visual Studio in administrator mode (regular command prompt may work but is untested; the x64 Native Tools Command Prompt will *not* work)
```
```suggestion
  - Install Visual Studio (the [free community edition](https://visualstudio.microsoft.com/vs/community/) works)
  - Open the x86 Native Tools Command Prompt that comes with Visual Studio in administrator mode (regular command prompt may work but is untested; the x64 Native Tools Command Prompt will *not* work)
```

vadi2 reviewed (commented):
What do you think of these improvements?

RollanzMushing committed:
Update src/mudlet-lua/tests/README.md

Co-authored-by: Vadim Peretokin <vperetokin@hey.com>

RollanzMushing committed:
Update src/mudlet-lua/tests/README.md

Co-authored-by: Vadim Peretokin <vperetokin@hey.com>

RollanzMushing committed:
Update README.md

- Corrected description of message that would appear if LuaRocks is not in `Path`
